modules/strongdm/helm.tf

```diff
@@ -0,0 +1,88 @@
+resource "helm_release" "gateway" {
+  count = local.enabled ? 2 : 0
```

Comment:
why 2? maybe make it a variable?

Reply:
The original gateway had 2 instances so I just reused the count logic.

Yes, that seems like a good idea to be able to change the number of gateways. I'll add that.